src/wombats/handlers/auth.clj

```diff
@@ -30,6 +30,13 @@
             {:headers {"Authorization" (str "token " access-token)
                        "Accept" "application/json"}}))
 
+(defn- remove-slash
+  "Removes the trailing slash from a url (if it exists)"
+  [url]
+  (if (.endsWith url "/")
+    (.substring url 0 (- (count url) 1))
```

Comment:
if you wanted to make this a pure function you could use `split` with `drop-last` and `join`

```clj
 (-> url
   (clojure.string/split #"") 
   (drop-last) 
   (clojure.string/join))
```

And you could pull in join and split so you don't have to use the fully qualified name.